src/EditorFeatures/CSharpTest/Interactive/NavigateTo/InteractiveNavigateToTests.cs

```diff
@@ -326,7 +326,7 @@ public void FindOverriddenMembers()
                 var unused = itemDisplay.Glyph;
 
                 Assert.Equal("Name", itemDisplay.Name);
-                Assert.Equal("type DogBed", itemDisplay.AdditionalInformation);
+                Assert.Equal($"{EditorFeaturesResources.Type}DogBed", itemDisplay.AdditionalInformation);
```

Comment:
Should it not? I'm not sure I'd trust localization to maintain that (if for no other reason than non-trimmed whitespace in XML is subtle), and it also means we can't use that string anywhere else we _don't_ want a space.